### DISCUSSION TITLE
I found a big scammer who plays employeer role, but don't pay for their work.

### DISCUSSION THREAD

**Discussion by happyeagle912** [community/community#118677] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, community!
This is andres from US.
I 'm here to announce that this github owner's big and mallicious scammer.
I worked for this github owner at dental-worker, dental-hire private repository.
But after 1 month, he never pay me and kicked me out.
I think this is very impolite way and too rude to mankind and insult to developers.
please inspector this github and block this github account.

https://github.com/miracle1125

Kind regards!
Thank you.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @happyeagle912 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
